package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "designer-bag-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "eslint": "^6.8.0",
+    "express": "^4.17.1",
+    "nodemon": "^2.0.3"
```

Comment:
eslint and nodemon should be devDependencies. Also you are missing pug which causes you app to error out.